server/utils/apTypeLabels.ts

```diff
@@ -1,12 +1,12 @@
 import { ApType } from '../@types/shared'
 
 export const apTypeLabels: Record<ApType, string> = {
-  normal: 'Standard AP',
+  normal: 'Standard (all AP types)',
   pipe: 'Psychologically Informed Planned Environment (PIPE)',
   esap: 'Enhanced Security AP (ESAP)',
-  rfap: 'Recovery Focused AP (RFAP)',
-  mhapElliottHouse: 'Specialist Mental Health AP (Elliott House - Midlands)',
-  mhapStJosephs: 'Specialist Mental Health AP (St Josephs - Greater Manchester)',
+  rfap: 'Recovery Focused AP',
```

Comment:
I think this does need to retain the initialism -- this wasn't in the original designs either, but I added it for consistency with regards to `Psychologically Informed Planned Environment (PIPE)` and `Enhanced Security AP (ESAP)`.